src/Models/Campaign.php

```diff
@@ -300,7 +361,8 @@ public function canBeCancelled(): bool
         // we can cancel campaigns that still have draft messages, because they haven't been entirely dispatched
         // a campaign that doesn't have any more draft messages (i.e. they have all been sent) cannot be cancelled, because the campaign is completed
 
-        if ($this->status_id === CampaignStatus::STATUS_SENT && $this->save_as_draft && $this->sent_count !== $this->messages()->count()) {
+        if ($this->status_id === CampaignStatus::STATUS_SENT && $this->save_as_draft && $this->sent_count !== $this->messages(
```

Comment:
indentation issue

Reply:
Reformatted this long if into multiple lines so it's easier to read and avoids that weirdness with the autoformatting.